truss/truss_handle/build.py

```diff
@@ -24,36 +28,62 @@
 
 
 def _populate_target_directory(
-    config: TrussConfig,
-    target_directory_path: Optional[str] = None,
-    template: str = "custom",
-    populate_dirs: bool = True,
+    config: TrussConfig, target_directory: str, python_dx: bool = False
```

Comment:
Similar to a comment below, this function is used in exactly one [location](https://github.com/basetenlabs/truss/blob/b8df3465a57dee1e8bc369f9833695136c56e19e/truss/truss_handle/build.py?plain=1#L84-L86) and never has alternative parameters for `template` or `populate_dirs`. I removed those arguments, and instead passed in `python_dx` which will be derived from the CLI.